scripts/verify-codegen.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
This needs to be hooked into CI to ensure we don't get out of sync. I plan to do that on a subsequent PR.